lib/dsl/validator.js

```diff
@@ -33,6 +33,7 @@ const ALPHABETIC = /^[A-Za-z]+$/;
 const ALPHABETIC_LOWER = /^[a-z]+$/;
 const ALPHANUMERIC = /^[A-Za-z][A-Za-z0-9]*$/;
 const ALPHANUMERIC_DASH = /^[A-Za-z][A-Za-z0-9-]*$/;
+const ALPHANUMERIC_LOWER = /^[a-z][a-z0-9]*$/;
```

Comment:
Underscore also needs to be allowed so add a specific pattern here. PACKAGE_NAME

Reply:
Renamed ALPHANUMERIC_LOWER to PACKAGE_NAME_PATTERN and added underscores to regex in second commit.